src/Highway.Data.EntityFramework/Contexts/DomainContext.cs

```diff
@@ -33,43 +28,5 @@ public DomainContext(T domain, ILog logger)
             : base(domain.ConnectionString, domain.Mappings, domain.Context, logger)
         {
         }
-
```

Comment:
All of this was moved into DataContext.  There is no need for the overrides or hiding the lower events.  With just the slightest finagling of DataContext.CommitAsync (making in async as well) I was able to just move this whole pattern down a level.  Since we're awaiting here, rather than passing the Task<int> further up the call stack, there is no material difference in behavior.  Do look at the updated methods in DataContext, though, just to make sure you agree.  It would be trivial to undo this little piece of work, and it isn't necessary for the Readonly work.